### PR TITLE
Add PaymentRef and StartDate fields to Get tenancy route

### PR DESCRIPTION
**Pull request by alexdesi**
- Fix property ref always being null

### CODE REVIEW

File: LBHTenancyAPI/Gateways/V1/UhTenanciesGateway.cs [hunk 1]
```diff
@@ -25,43 +25,45 @@ public List<TenancyListItem> GetTenanciesByRefs(List<string> tenancyRefs)
...
-                    tenagree.cur_bal as CurrentBalance, 
-                    tenagree.prop_ref as PropertyRef, 
-                    tenagree.tenure as Tenure, 
-                    tenagree.rent as Rent, 
-                    tenagree.service as Service, 
-                    tenagree.other_charge as OtherCharge, 
+                  @"SELECT
+                    tenagree.tag_ref as TenancyRef,
+                    tenagree.u_saff_rentacc as PaymentRef,
+                    tenagree.cot as StartDate,
```
Comment (mrosel) [inline LBHTenancyAPI/Gateways/V1/UhTenanciesGateway.cs]: nice find!

File: LBHTenancyAPI/UseCases/V1/TenancyDetailsForRef.cs [hunk 2]
```diff
@@ -20,28 +21,16 @@ public TenancyResponse Execute(string tenancyRef)
...
-                    Rent = tenancyResponse.Rent.ToString("C"),
-                    Service = tenancyResponse.Service.ToString("C"),
-                    OtherCharge = tenancyResponse.OtherCharge.ToString("C"),
-                    PrimaryContactName = tenancyResponse.PrimaryContactName,
-                    PrimaryContactLongAddress = tenancyResponse.PrimaryContactLongAddress,
-                    PrimaryContactPostcode = tenancyResponse.PrimaryContactPostcode,
-                    ArrearsActionDiary = new List<ArrearsActionDiaryEntry>(),
-                    ArrearsAgreements = new List<ArrearsAgreement>()
-                };
+                response.TenancyDetails = new Tenancy();
```
Comment (mrosel) [inline LBHTenancyAPI/UseCases/V1/TenancyDetailsForRef.cs]: Would it be too complicated at this point to return a 404 instead?

File: Makefile [hunk 3]
```diff
@@ -13,7 +13,7 @@ setup: docker-build
 
 .PHONY: serve
 serve:
-	docker run -p 3000:80 -it --rm lbhtenancyapi
+	docker-compose run -p 3000:80 -e UH_URL=$(TEST_UH_CONNECTION_STRING) --rm lbhtenancyapi
```
Comment (mrosel) [inline Makefile]: :tada: one step closer to real dev/test env's

File: LBHTenancyAPI/UseCases/V1/TenancyDetailsForRef.cs (hunk 2 above)
Comment (cob16) [reply LBHTenancyAPI/UseCases/V1/TenancyDetailsForRef.cs]: The change is larger than just the use case. I stated it before and it got out of hand.

**Pull request merged**